tests/macros.rs

```diff
@@ -50,6 +50,20 @@ macro_rules! error {
     };
 }
 
+/// Verify that the input does not cause a panic.
+/// Useful for when testing input found through fuzzing.
+#[macro_export]
+macro_rules! no_panic {
```

Comment:
Isn't the default test already means no panic?

Reply:
I wanted to make clear that this macro verifies nothing, it doesn't check if the outcome is `Ok` or `Err`, only that it doesn't panic. Which is indeed normal test behavior, so the macro only calls the function with the default options and nothing goes on inside.